x-pack/plugins/observability_solution/synthetics/server/synthetics_service/private_location/synthetics_private_location.ts

```diff
@@ -369,7 +374,10 @@ export class SyntheticsPrivateLocation {
       return await this.server.fleet.packagePolicyService.bulkCreate(
         soClient,
         esClient,
-        newPolicies
+        newPolicies,
+        {
+          packageInfos: this.packageInfo ? [this.packageInfo] : undefined,
```

Comment:
Infos isn't really the right plural for info. It should be information
```suggestion
          packageInformation: this.packageInfo ? [this.packageInfo] : undefined,
```